lib/pkgcloud/openstack/network/client/securityGroupRules.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * securityGroupRules.js: Instance methods for working with security group rules
+ * for Openstack Networking
+ *
+ * (C) 2015 Rackspace
+ *      Shaunak Kashyap
+ * MIT LICENSE
+ */
+
+var urlJoin = require('url-join');
+
+var securityGroupRulesResourcePath = '/security-group-rules';
+
+// Declaring variables for helper functions defined later
+var _convertSecurityGroupRuleToWireFormat;
+
+/**
+ * client.getSecurityGroupRules
+ *
+ * @description get the list of security group rules for an account
+ *
+ * @param {object|Function}   options
+ * @param {Function}          callback
+ */
+exports.getSecurityGroupRules = function (options, callback) {
+  var self = this;
+
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+
+  var getSecurityGroupRuleOpts = {
+    path: securityGroupRulesResourcePath
+  };
+
+  this._request(getSecurityGroupRuleOpts, function (err, body) {
+    if (err) {
+      return callback(err);
+    }
+    else if (!body || !body.security_group_rules || !(body.security_group_rules instanceof Array)) {
+      return new Error('Malformed API Response');
+    }
+
+    return callback(null, body.security_group_rules.map(function (securityGroupRule) {
```

Comment:
Until we audit more broadly, I'd prefer `callback(err, data)` as that matches the convention I've been doing for 2 years.